src/Controller/DefaultController.php

```diff
@@ -2266,6 +2287,7 @@ public function manageParametersAction(Request $request)
                     $t->trans('Parameters updated', array(), 'BinovoElkarBackup')
                 );
             }
+            $this->clearCache();
             $result = $this->redirect($this->generateUrl('manageParameters'));
```

Comment:
A long shot about why we need the sleep:

redirect seems to send an HTTP redirect and it seems that if execution is fast enough the deletion is not still complete and the new request sees the old parameters. That seems weird and suggests some kind of race condition at "file system" level but that's what it seems to be.
In PHP 8.1 they have just added Fsync (https://php.watch/versions/8.1/fsync-fdatasync) that might have solved the issue but that's another story.

It seems that there is a way of forwarding a request instead of redirecting: https://symfony.com/doc/4.4/controller/forwarding.html

This is quite counterintuitive but who knows, maybe if you are in the exact same request/process that have just deleted the cache it sees the new one.

Reply:
This doesn't work, it gets the old parameters (I think they're already loaded from cache).